modules/git/command.go

```diff
@@ -40,19 +40,17 @@ const DefaultLocale = "C"
 
 // Command represents a command with its subcommands or arguments.
 type Command struct {
-	name             string
 	args             []string
 	parentContext    context.Context
 	desc             string
+	extraDesc        string
```

Comment:
Why?

Reply:
To keep the default description, so we can remove most descriptions when invoking git command.

Reply:
Then `SetDescription` will be removed, right?

Reply:
> 

That still will be kept but in almost places, we don't need that function.

Reply:
`SetDescription still will be kept but in most places`

Then I still can not get the point why `SetDescription` and `SetExtraDescription` should both exist.

Reply:
After grepping the calls to `SetDescription` and `SetExtraDescription`. If I understand correctly, the `SetDescription` is mainly for making the description of git command with long arguments to be concise.

But `SetDescription` and `SetExtraDescription` really seems similar and may be misused later ....

Reply:
Just thought about an interesting idea:

`cmd.SetDescription("${cmd}, and other message: %s, %d", filename, line)`

The `${cmd}` could be replaced by the command and arguments.

Then only one function is enough. And one command always have one description (no `extra`). And no need to call `Sprintf` separately when setting the description.


ps: I know the trick `${cmd}` is arguable, so not sure whether it is good enough.